docs/training/apis-howto/overview.rst

```diff
@@ -50,19 +50,19 @@ experiment is controlled by the container image that has been configured for tha
 Determined provides prebuilt Docker images that include TensorFlow 2.8, 1.15, 2.5, 2.6, and 2.7,
 respectively:
 
--  ``determinedai/environments:cuda-11.3-pytorch-1.10-tf-2.8-gpu-0.19.4`` (default)
--  ``determinedai/environments:cuda-10.2-pytorch-1.7-tf-1.15-gpu-0.19.4``
--  ``determinedai/environments:cuda-11.2-tf-2.5-gpu-0.19.4``
--  ``determinedai/environments:cuda-11.2-tf-2.6-gpu-0.19.4``
--  ``determinedai/environments:cuda-11.2-tf-2.7-gpu-0.19.4``
+-  ``determinedai/environments:cuda-11.3-pytorch-1.10-tf-2.8-gpu-0.19.10`` (default)
+-  ``determinedai/environments:cuda-10.2-pytorch-1.7-tf-1.15-gpu-0.19.10``
+-  ``determinedai/environments-dev:cuda-11.2-tf-2.5-gpu-0.19.10``
```

Comment:
We should probably remove the mentions of tf-2.5 and tf-2.6 but I prefer to do it in a separate PR.